test/explain/algorithm/test_captum_explainer.py

```diff
@@ -142,9 +174,31 @@ def test_captum_hetero_data(node_mask_type, edge_mask_type, index, hetero_data,
 
     model = hetero_model(hetero_data.metadata())
 
+    kwargs = {}
+    if method == 'Occlusion':
+        if edge_mask_type is None:
+            sliding_window_shapes = ((3, 3), (3, 3))
+        elif node_mask_type is None:
+            sliding_window_shapes = ((5, ), (5, ), (5, ))
+        else:
+            sliding_window_shapes = ((3, 3), (3, 3), (5, ), (5, ), (5, ))
+        kwargs['sliding_window_shapes'] = sliding_window_shapes
```

Comment:
As `Occlusion` is an unsupported method, we do not need those lines, correct?

Reply:
Yeah that true removed them.